extensions/adobe/experience/decisioning/decisionitem.schema.json

```diff
@@ -69,6 +69,11 @@
           "$ref": "https://ns.adobe.com/experience/decisioning/calendar-constraint-details",
           "title": "Item Calendar Constraints",
           "description": "Defines the date range during which this decision item is eligible. This constraint may not be strictly enforced always."
+        },
+        "xdm:_trackingToken": {
```

Comment:
missed to notice the path -  let's double check this.

Reply:
The team decided to change the path to be consistent with all other attributes. I have updated it to "https://ns.adobe.com/experience/decisioning/decisionitem/trackingToken"